### PR TITLE
document: adapt detail view

### PR DESCRIPTION
**Pull request by AoNoOokami**
* Adds 'Get', 'Online access' and 'Description' tabs.
* Moves fields according to mockup.
* Moves 'Add' button to the right.

Co-Authored-by: Alicia Zangger <alicia.zangger@rero.ch>

## Why are you opening this PR?

Implements task 1447 of US 1305 (adapt detailed view in professional interface)
https://tree.taiga.io/project/rero21-reroils/us/1305?milestone=263424

## How to test?

Create a record with the fields that you want to check the display.

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Extracted translations?

### CODE REVIEW

**Comment by benerken** [comment]
Nice job ! 

Do you need to enable the tab "Online Access" when there is no online access available for the document ? **I suggest to disable this tab if there is nothing to display (if the PO agree).** 

image

**Comment by AoNoOokami** [comment]
> Nice job !
> 
>     1. Do you need to enable the tab "Online Access" when there is no online access available for the document ? **I suggest to disable this tab if there is nothing to display (if the PO agree).**
> 
> 
> image
> 
>     1. The labels are missing no ? (like Authors, abstracts etc...)
>        (Are the labels in the mockup ? I have'nt check that)
>        We find it's clearer with labels for all fields but it's our PoV.

1. Good suggestions, I modified the view accordingly.
2. According to the mockup, the labels are only in the tabs.

File: projects/admin/src/app/record/detail-view/document-detail-view/document-detail-view.component.html [hunk 1]
```diff
@@ -16,394 +16,442 @@
...
+    </ng-container>
 
     <!-- TITLE -->
-    <ng-container *ngIf="record.metadata.title">
-      <h1 *ngIf="record.metadata.title | mainTitle as title" class="pl-0">
+    <ng-container *ngIf="record.metadata.ui_title_text_responsibility; else simpleTitleBlock">
+        <h2>{{ record.metadata.ui_title_text_responsibility }}</h2>>
+    </ng-container>
+    <ng-template #simpleTitleBlock>
+      <h1 *ngIf="record.metadata.title && record.metadata.title | mainTitle as title" class="pl-0">
```
Comment (sebastiendeleze) [inline projects/admin/src/app/record/detail-view/document-detail-view/document-detail-view.component.html:31]: Is it really a H1, not a H2?

Comment (AoNoOokami) [reply projects/admin/src/app/record/detail-view/document-detail-view/document-detail-view.component.html]: Indeed

**Pull request merged**